anchor/common/qbft/src/lib.rs

```diff
@@ -432,7 +468,11 @@ where
             debug!(hash = ?valid_data.hash, "Current leader proposing data");
 
             // Send the initial proposal and then the following prepare
-            self.send_proposal(valid_data.hash, valid_data.data.expect("Start data exists"));
+            if let Err(e) =
+                self.send_proposal(valid_data.hash, valid_data.data.expect("Start data exists"))
```

Comment:
@dknopik @Zacholme7 should we get the release branch merged here? The expect has been removed